configuration/renderer.toml

```diff
@@ -22,6 +22,7 @@ minor = 0
 patch = 0
 
 [application.window]
+mode = 0
```

Comment:
I would advise to use a string as descriptor. It will be a pain to look up the coresponding values everytime.